### DISCUSSION TITLE
Proxy not used for custom client build request (Docker)

### DISCUSSION THREAD

**Discussion by JohnnyHenver** [rustdesk/rustdesk-server-pro#301] [Q&A] [answered]
I am glad that 1.3.10 added proxy support for license verification. However I see that the build process for custom clients does not honour his setting or other typical ways to configure http_proxy/https_proxy in docker containers.

Please add proxy support for these outgoing connections as well.

**Comment by rustdesk** [comment]
Thanks, will be in next version.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk-server-pro/releases/tag/1.3.12

**Reply by JohnnyHenver** [reply]
@rustdesk was this removed in 1.4.0? Custom build seems to ignore settings in .env-file after update from 1.3.12 to 1.4.0.

**Reply by rustdesk** [reply]
Nothing change here
